src/cluster/OnLoop.h

```diff
@@ -38,12 +41,15 @@ class OnLoopProcess : public zeek::iosource::IOSource {
      *
      * @param proc The instance processing.
      * @param tag The tag to use as the IOSource's tag.
+     * @param max_queue_size How many messages to queue before blocking the producing thread.
+     * @param cond_timeout If a producer is blocked for more than that many microseconds, report a warning.
+     * @param main_thread_id The ID of the main thread for usage checks.
```

Comment:
I am wondering whether it is worth introducing this "rich" policy. An alternative could be to just disconnect the client and force it to reconnect and reconcile its state (we should still count such disconnects).